marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -447,27 +396,23 @@
     // Overlay properties in front of Camera 
     // Arguments: position in front of Camera, overlay dimensions, overlay image url
     function getInFrontOverlayProperties(positionInFront, dimensions, url) {
-
-        var index = MyAvatar.getJointIndex("Head");
-
         return {
             position: Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, positionInFront)),
             rotation: Camera.orientation,
             parentID: MyAvatar.sessionUUID,
-            parentJointIndex: index,
+            parentJointIndex: MyAvatar.getJointIndex("_CAMERA_MATRIX"),
             dimensions: dimensions,
             url: url,
             ignoreRayIntersection: false,
             drawInFront: true,
```

Comment:
does this property still work? i thought you had to use `renderLayer`, no?

Reply:
If it were a local entity, then yes. Since it's an overlay I don't need to.